lib/query/querycompiler.js

```diff
@@ -199,10 +199,21 @@ class QueryCompiler {
     );
   }
 
+  _optimizerHints() {
+    let hints = this.grouped.optimizerHints || [];
+    hints = hints.map((hint) => compact(hint.value).join(' '));
+    hints = compact(hints).join(' ');
+    if (hints) {
```

Comment:
can be a ternary statement

Reply:
@kibertoad Sure I can change it, I just lookend arround the codebase, and it seems that

```
if (...) return ...
return ...
```

pattern is much more frequent than

```
return ... ? ... : ....
```
pattern. So I followed the more common style.

Should I change it?

Reply:
codebase, unfortunately, is full of suboptimal code, definitely don't follow it :)

Reply:
but either is fine, pick the one that looks most readable to you

Reply:
changed in 8414e94